docs/configuration.md

```diff
@@ -1752,6 +1752,13 @@ showDF(properties, numRows = 200, truncate = FALSE)
     Executable for executing R scripts in client modes for driver. Ignored in cluster modes.
   </td>
 </tr>
+<tr>
+  <td><code>spark.r.shell.command</code></td>
+  <td>R</td>
+  <td>
+    Executable for executing R shell.
```

Comment:
"Executable for sparkR shell"?
"Executable for R when running the sparkR shell"?

(the sparkR shell command starts with lower case `s`)